app/ags/build.sbt

```diff
@@ -46,8 +50,10 @@ def common(version: Version) = AppConfig(
     "org.osgi.framework.bootdelegation"                -> "*",
     "org.osgi.framework.startlevel.beginning"          -> "99",
     "org.osgi.framework.storage.clean"                 -> "onFirstInit",
-    "org.osgi.service.http.port"                       -> "9123",
-    "org.osgi.service.http.secure.enabled"             -> "false"
+    "org.osgi.service.http.port"                       -> "8442",
```

Comment:
I guess this is the core change, is it?

Reply:
Along with adding the `pax-web` bundles, linking in the `gemKeystore`, and updating the ocs credentials file.